crmsh/sbd.py

```diff
@@ -378,11 +381,12 @@ def configure_sbd_resource(self):
         """
         if not utils.package_is_installed("sbd") or \
                 not utils.service_is_enabled("sbd.service") or \
-                utils.has_resource_configured("stonith:external/sbd"):
+                utils.has_resource_configured(self.SBD_RA_TYPE):
             return
 
         if self._get_sbd_device_from_config():
-            if not bootstrap.invokerc("crm configure primitive stonith-sbd stonith:external/sbd pcmk_delay_max=30s"):
+            pcmk_delay_max_str = " {}={}s".format(self.PCMK_DELAY_MAX, self.PCMK_DELAY_MAX_VALUE) if utils.is_two_nodes_cluster() else ""
+            if not bootstrap.invokerc("crm configure primitive stonith-sbd {}{}".format(self.SBD_RA_TYPE, pcmk_delay_max_str)):
```

Comment:
I do prefer to let adjust_pcmk_delay_max() be idempotent, then it can be called any time and do the adjustment accordingly, not matter stage, join, or remove. 

For 3->2, sorry, I didn't describe completely. My brain extended to the rare use case when users manually configure pcmk_delay_max on their own before 'remove' node. I agree this is the rare situation.